CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,65 @@
+
+# Otrego Code of Conduct
+
+## Our Values
+
+Clear communication is foundational for healthy collaboration. We value the time needed to be thoughtful in how we communicate and value the time contributors offer to the project. We make efforts to counter prejudice and biases against contributors and contributions. Ultimately, we put personal development ahead of production.
+
+As representatives of the go community, we strive for a welcoming environment. We aim to promote growth of the game and expand public knowledge of go, from the board to go’s intersections with technology and self-knowledge.
```

Comment:
Peer Review:  Willingness to volunteer time to test collegues code.
                      Willingness to openly show collegues ones own code.  
Collaboration:  
Mentoring:

Reply:
```suggestion
Clear communication is foundational for healthy collaboration. We value the time needed to be thoughtful in how we communicate and value the time contributors offer to the project. We make efforts to counter prejudice and biases against contributors and contributions, and prioritize transparency in code and code reviews. Ultimately, we put personal development ahead of production.

A culture of collaboration means valuing our contributors' different backgrounds and distinct contributions. We encourage reaching out to offer assistance and taking opportunities for mentorship that arise.

As representatives of the go community, we strive for a welcoming environment. We aim to promote growth of the game and expand public knowledge of go, from the board to go’s intersections with technology and self-knowledge.
```

Thanks, Brian. I added a paragraph on mentorship and collaboration, and I think your point about transparency is important and added a line to the first paragraph to emphasize that as well. What do you think?